src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Colors[] colors = Colors.values();
+        int i = new Random().nextInt(colors.length);
```

Comment:
move random to the class constant, like:
![image](https://user-images.githubusercontent.com/12864128/221858172-2280d56e-200c-4b5e-b8f9-83f6643c6358.png)